Minigames/src/main/java/au/com/mineauz/minigames/minigame/Minigame.java

```diff
@@ -197,7 +194,7 @@ private void setup(MinigameType type, Location start) {
         addConfigFlag(displayName);
         addConfigFlag(enableFlight);
         addConfigFlag(enabled);
-        addConfigFlag(endPosition);
+        addConfigFlag(endLocation);
```

Comment:
Is it backwards compatible?

Reply:
since this touches just the private name of the ConfigFlag everything should be fine. The save file still uses "endpos" (until a DataFixerUpper will get written) The only break this does is for Plugins that may depend on Minigames.
This should only be Regions the comapion as far as I know of.